admin/spec/service/generate-pins.service.spec.js

```diff
@@ -101,20 +104,21 @@ describe('generate-pins.service', () => {
         })
       })
       it('when pin in empty', async (done) => {
```

Comment:
should that be 'is' empty?

Reply:
@GuyHarwood cheers for spotting that, description is updated